packages/fx-core/resource/package.nls.json

```diff
@@ -503,8 +452,6 @@
   "core.AzureResourceKeyVaultNewUI.detail": "A cloud service for securely storing and accessing secrets",
   "core.SingleSignOnOption.label": "Single Sign-On",
   "core.SingleSignOnOption.detail": "Develop a Single Sign-On feature for Teams Launch pages and Bot capability",
-  "core.ApiConnectionOption.label": "API Connection",
```

Comment:
Is this used by how to guide?

Reply:
how to guide uses the string from vscode extension package
![image](https://github.com/OfficeDev/TeamsFx/assets/16380704/e56a7714-e2cc-4d8e-beaa-8d30a4b0e8d7)